### PR TITLE
WheelPicker + SectionsWheelPicker - RTL support

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Add RTL support (and disableRTL support) to WheelPicker and SectionsWheelPicker.
Also added an example to the SectionsWheelPicker example screen to show this new feature on RTL. (and to make sure it works 😇) 
This PR is an infrastructure to support WOAUILIB-3027

## Changelog
Add RTL support (and disableRTL support) to WheelPicker and SectionsWheelPicker.

### CODE REVIEW

File: src/incubator/WheelPicker/index.tsx [hunk 1]
```diff
@@ -109,6 +119,9 @@ const WheelPicker = ({
   const scrollHandler = useAnimatedScrollHandler(e => {
     offset.value = e.contentOffset.y;
   });
+  const shouldDisableRTL = useMemo(() => {
+    return Constants.isRTL && disableRTL;
+  }, [disableRTL]);
```
[the remark this answers is not in the document]
Comment (lidord-wix) [reply src/incubator/WheelPicker/index.tsx:124]: I added `shouldDisableRTL` to `SectionsWheelPicker` as well.. the idea is to let the user pass the `disableRTL` prop always and not only for RTL languages (we should take care of the RTL check).
And I don't really like the idea of renaming the prop in `Item`, I prefer to keep all the props with the same name - `disableRTL`

**Comment by lidord-wix** [comment]
@M-i-k-e-l I added an example to the screen like in the WheelPickerScreen but kept the previous example (mainly for us, to see how it changes and not changes in RTL).
let me know what you think

**Review by M-i-k-e-l — APPROVED** [review]
Approving with these comments:
1. I'd probably put the `Disable RTL` next to the bottom example since it's more relevant to that.
2. If you scroll the hours in the dialog you'll see a small adjustment in the minutes side, this happens because there is a difference in the width of `1` and `10`, I think this is a real bug in our component because if a user will want to do something like this they will have this bug (I did not find a way to solve it) - perhaps we want to open a ticket for this.

**Comment by lidord-wix** [comment]
@M-i-k-e-l I updated the screen with Diana's design, feel free to go over it if you want and merge it if not :)

**Comment by lidord-wix** [comment]
> > @M-i-k-e-l I updated the screen with Diana's design, feel free to go over it if you want and merge it if not :)
> 
> Why not both? :) The button is disabled when not in RTL 👏

All the credit to Diana :)

**Pull request merged**